### DISCUSSION TITLE
非admin用户令牌功能

### DISCUSSION THREAD

**Discussion by zch11230** [AlistGo/alist#4597] [Ideas]
用POST实现一些自己想要的功能，admin用户有令牌功能,就可以免登陆直接POST，但权限太高，想其它用户也可以用令牌功能，谢谢。

**Comment by 520yxl** [comment]
https://github.com/alist-org/alist/discussions/2501

**Comment by 520yxl** [comment]
有登陆接口

**Reply by zch11230** [reply]
谢谢回复，目前就是用的登陆拿token方式，要考虑token有效期问题，和失效时的处理，想着如果有令牌的话，就简单一些。
